client/components/product-card/options.jsx

```diff
@@ -29,7 +33,14 @@ const ProductCardOptions = ( {
 	return (
 		<Fragment>
 			{ ! hideRadios && optionsLabel && (
-				<h4 className="product-card__options-label">{ optionsLabel }</h4>
+				<h4 className="product-card__options-label">
+					{ optionsLabel }
+					<InfoPopover position="right">
+						{ translate(
+							'Records are all posts, pages, custom post types, and other types of content indexed by Jetpack Search.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Records are all posts, pages, custom post types and other types of content indexed by Jetpack Search.' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/Search-plan-card-add-info-icon)